<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClientWindow</class>
 <widget class="QMainWindow" name="ClientWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>544</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>120</width>
      <height>450</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Username:</string>
     </property>
    </column>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>600</width>
      <height>50</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QFontComboBox" name="fontCbx"/>
     </item>
     <item>
      <widget class="QComboBox" name="sizeCbx">
       <item>
        <property name="text">
         <string>5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>7</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="boldTBtn">
       <property name="toolTip">
        <string>Bold</string>
       </property>
       <property name="text">
        <string>B</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italicTbtn">
       <property name="toolTip">
        <string>Incline</string>
       </property>
       <property name="text">
        <string>I</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="underlineTBtn">
       <property name="toolTip">
        <string>Underline</string>
       </property>
       <property name="text">
        <string>U</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="colorTBtn">
       <property name="toolTip">
        <string>Colour</string>
       </property>
       <property name="text">
        <string>Col</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="FileTBtn">
       <property name="text">
        <string>F</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearTBtn">
       <property name="toolTip">
        <string>Clear</string>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="msgTextEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>600</width>
      <height>100</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>600</width>
      <height>300</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/Images/Naruto.jpg);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>602</width>
      <height>30</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="spacing">
      <number>81</number>
     </property>
     <property name="leftMargin">
      <number>81</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>81</number>
     </property>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string> Member Online:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exitButton">
       <property name="text">
        <string>exit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="Plabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-160</y>
      <width>731</width>
      <height>770</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>Plabel</zorder>
   <zorder>tableWidget</zorder>
   <zorder>frame</zorder>
   <zorder>msgTextEdit</zorder>
   <zorder>splitter</zorder>
   <zorder>layoutWidget</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuClient">
    <property name="title">
     <string>Client</string>
    </property>
   </widget>
   <addaction name="menuClient"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
